backend/src/middleware/userMiddleware.js

```diff
@@ -11,6 +13,10 @@ export default {
       return result
     },
     UpdateUser: async (resolve, root, args, context, info) => {
```

Comment:
This should go into a middleware, not a resolver. Ask @mattwr18 where he placed the validation for `SocialMedia`. We could have a nice `middleware/validations/index.js` where we place all data validations.

If you do it that way, you can also re-use the same validation method for `CreateUser` and `UpdateUser`.

Reply:
Do you refere to this? Or are there some other changes somewhere
![image](https://user-images.githubusercontent.com/1238238/56420003-98d49680-629c-11e9-9da5-a6a8a5be3a68.png)
`middleware/validations/index.js` is not existent in backend master
@roschaefer @mattwr18

Reply:
Oh yes! Apparently this got overlooked. Either we use [constraint directives](https://github.com/confuser/graphql-constraint-directive) or we implement all data validations inside a middleware.

@ulfgebhardt could you give the constraint directives a try? You are the GraphQL expert and @mattwr18 had a hard time when he gave it a shot.